examples/pes/pswater/input.xml

```diff
@@ -18,7 +19,7 @@
   <system>
     <initialize nbeads='1'> 
       <file mode='xyz' units='angstrom'> h2o.xyz </file>
```

Comment:
The idea is that since the fastxyz we should avoid specifying the units inside the i-pi input. The basic reason is that now it is possible to specify the units directly into the xyz file... so if nothing is specified then atomic units are assumed... we should continue this discussion at #151